ui/app/controllers/vault/cluster/clients/config.js

```diff
@@ -0,0 +1,3 @@
+import Controller from '@ember/controller';
```

Comment:
Need a controller to use the `loading` action in the corresponding route

Reply:
When I remove it I get this error - perhaps I'm missing something?

<img width="1387" alt="Screen Shot 2022-02-07 at 10 28 09 AM" src="https://user-images.githubusercontent.com/68122737/152849319-272abc38-704a-4298-b40f-dd7fc3ed6ab4.png">

Reply:
Hmm that's interesting. There's a comment in the docs to do a check for the controller instance in case it hasn't been instantiated yet.

https://api.emberjs.com/ember/4.1/classes/Route/events/loading?anchor=loading

Reply:
Oh! I'll look into this - thanks for sending